databuilder/dsl.py

```diff
@@ -235,6 +241,13 @@ def __init__(self, row: Row):
         """
         self.row = row
 
+    @classmethod
+    def from_contract(cls, contract):
+        qm_table = Table(contract._name)
+        # TODO: revisit this!  As things stand, this will generate SQL with an
+        # unnecessary PARTITION OVER, which may carry a performance penalty.
```

Comment:
Since this may be important, could you create a story to cover this rather than leaving it as a TODO?

Reply:
See https://app.shortcut.com/ebm-datalab/story/569/decide-how-to-instantiate-patienttable.